File: wear/src/main/java/com/codegy/aerlink/utils/ServiceUtils.java
package com.codegy.aerlink.utils;

import android.app.Notification;
import com.codegy.aerlink.connection.command.Command;

/**
 * Created by Guiye on 19/5/15.
 */
public interface ServiceUtils {

    boolean isAerlinkAvailable();
    void addCommandToQueue(Command command);
    void notify(String tag, int id, Notification notification);
    void cancelNotification(String tag, int id);

}

File: wear/src/main/java/com/codegy/aerlink/utils/ServiceUtils.kt
package com.codegy.aerlink.utils

import android.app.Notification
import com.codegy.aerlink.connection.Command

interface ServiceUtils {
    fun addCommandToQueue(command: Command)
    fun notify(tag: String? = null, id: Int, notification: Notification)
    fun cancelNotification(tag: String? = null, id: Int)
}